doc/source/appendix/benchmarks.rst

```diff
@@ -197,7 +199,7 @@ supports the following options:
 * ``--nqubits`` (``int``): Number of qubits in the circuit.
 * ``--nangles`` (``int``): Number of variational parameters in the QAOA ansatz. The parameters are initialized according to uniform distribution in [0, 0.1].
 * ``--trotter`` (``bool``): If ``True`` it uses the Trotter decomposition to apply the exponential operators.
-* ``--solver`` (``str``): :ref:`Solver <Solvers>` to use for applying the exponential operators.
+* ``--solver`` (``str``): :ref:`Solver <Optimizers>` to use for applying the exponential operators.
```

Comment:
I think we have renamed Solvers in Optimizers several months ago, however these scripts are using the old notation.

Reply:
I'm a bit confused, here with solvers we intend the methods in ``solver.py`` like `exp`, `rk4`, `rk45` or minimization methods like `cma`, `sgd`, `Powell` etc?
Because the former are not even described in the API reference, while the latter are already in line 203.